static/js/place/chart.tsx

```diff
@@ -114,12 +130,26 @@ class Chart extends React.Component<ChartPropType, ChartStateType> {
     const dateString = this.props.snapshot
       ? "(" + this.props.snapshot.date + ")"
       : "";
-    const sources = this.props.trend
-      ? this.props.trend.sources
-      : this.props.snapshot.sources;
-    const exploreUrl = this.props.trend
-      ? this.props.trend.exploreUrl
-      : this.props.snapshot.exploreUrl;
+    let sources = [];
+    if (this.props.chartType !== chartTypeEnum.CHOROPLETH) {
```

Comment:
does choropleth data have sources now?

Reply:
not yet, that's something I still need to add. Also need to get the exploreURL when getting choropleth data